requirements-base.txt

```diff
@@ -71,7 +71,7 @@ sentry-ophio==1.0.0
 sentry-protos==0.1.72
 sentry-redis-tools>=0.5.0
 sentry-relay>=0.9.8
-sentry-sdk[http2]>=2.25.1
+sentry-sdk[http2]>=2.27.0
```

Comment:
this doesn't actually matter -- this just sets the lower bound -- what we install from is always the frozen files (aka this PR isn't needed)

Reply:
Easy close.  Thanks.